app/models/concerns/simple_state_machine.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+# SimpleStateMachine module provides a lightweight alternative to the AASM gem
+module SimpleStateMachine
+  # Extends the including class with class methods
+  def self.included(base)
+    base.extend(ClassMethods)
+    base.class_attribute :aasm_column, :aasm_states, :aasm_events, :aasm_initial_state, instance_accessor: false
```

Comment:
Given that this is now SSM, maybe we should go ahead and rename these methods?
```suggestion
    base.class_attribute :ssm_column, :ssm_states, :ssm_events, :ssm_initial_state, instance_accessor: false
```